cmd/openshift-install/graph.go

```diff
@@ -42,7 +44,7 @@ func runGraphCmd(cmd *cobra.Command, args []string) error {
 		string(gographviz.Shape): "box",
 		string(gographviz.Style): "filled",
 	}
-	for _, t := range targets {
+	for _, t := range cmdTargets {
```

Comment:
What happens to targets? This appears to throw it out.

Reply:
It's now passed as an argument on l. 30.